cmds/exp/crc/crc.go

```diff
@@ -47,32 +44,43 @@ func main() {
 		"crc64-iso":        crc64.New(crc64.MakeTable(crc64.ISO)),
 	}
 
-	h, ok := functions[*function]
+	h, ok := functions[function]
 	if !ok {
-		expected := ""
-		for k := range functions {
-			expected += " " + k
+		var k []string
+		for key := range functions {
+			k = append(k, key)
 		}
-		log.Fatalf("invalid function %q, expected one of:%s", *function, expected)
+		return fmt.Errorf("invalid function %q, expected one of: %s", function, strings.Join(k, " "))
```

Comment:
would it make sense to wrap os.ErrInvalid error so that your tests could use errors.Is

Reply:
Yes, this is a good idea and we use os.ErrInvalid only once in cmds/core/* but I think we have more cases of wrong arguments :)